### PR TITLE
Load testdata into database for CI

### PR DESCRIPTION
**Pull request by FinnAtAsseco**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -17,7 +17,11 @@ services:
...
-        build:
-            DCSA-Information-Model/datamodel/
+        image: postgres:10-alpine
         ports:
             - 5432:5432
+        volumes:
+          - ./DCSA-Information-Model/datamodel/initdb.d:/docker-entrypoint-initdb.d:ro
+          - ./DCSA-Information-Model/datamodel/referencedata.d:/referencedata.d:ro
+          - ./DCSA-Information-Model/datamodel/samples.d:/samples.d:ro
+          - ./DCSA-Information-Model/datamodel/testdata.d:/testdata.d:ro
```
Comment (nt-as) [inline docker-compose.yml:27]: I think this should go into the docker file in the IM.

File: .github/workflows/dev.yml [hunk 2]
```diff
@@ -43,10 +43,16 @@ jobs:
...
+        run: cd DCSA-EBL && mvn -B package -Ddcsa.events.version=$DCSA_EVENT_CORE_PROJECT_VERSION -Ddcsa.event.tag=dev-ebl -Ddcsa.artifacttype=-SNAPSHOT #Need to skip the tests as they depend on a running DB
 
-      - name: Build and run the microservices
-        run: cd DCSA-EBL && docker-compose -f docker-compose.yml up -d -V
+      - name: Start the database
+        run: cd DCSA-EBL && docker-compose -f docker-compose.yml up -d -V dcsa-test-db
+
+      - name: Load test data into the database
+        run: sleep 3 && docker exec -i dcsa_db sh -c "cat /testdata.d/* | psql -h localhost -U postgres postgres"
+
```
Comment (nt-as) [inline .github/workflows/dev.yml:53]: Maybe it is time we stop doing this outside our docker image and instead have it load test data on init (possibly conditionally via an ENV var).  I am getting tired of all the work arounds and issues this causes for our CI.

I would be in favor of overturning our previous decision that the image should only contain non-test data.

File: docker-compose.yml (hunk 1 above)
Comment (FinnAtAsseco) [reply docker-compose.yml:27]: I prefer to avoid making a "special" database image just for some tables and some data.

Comment (nt-as) [reply docker-compose.yml:27]: The docker file already includes the first 3, so they are redundant.  As for the last one, we can also load that externally like we do in all the other CIs.

File: .github/workflows/dev.yml (hunk 2 above)
Comment (FinnAtAsseco) [reply .github/workflows/dev.yml:53]: I guess we need to have a discussion in the team about how to move forward so we are aligned.

File: docker-compose.yml (hunk 1 above)
Comment (FinnAtAsseco) [reply docker-compose.yml:27]: They would be redundant if I was using the IM image. But this PR changes that to use a pure postgresql image so we don't have to build and rebuild the image all the time.

**Pull request closed without merging**